tools/js-tools/load-eslint-ignore.js

```diff
@@ -1,7 +1,7 @@
 const fs = require( 'fs' );
 const path = require( 'path' );
 const debugload = require( 'debug' )( 'load-eslint-ignore:load' );
-const debugrp = require( 'debug' )( 'load-eslint-ignore:reverse-prefix' );
+const debugrp = require( 'debug' )( 'load-eslint-ignore:rule-process' );
```

Comment:
Is this intended to be `rule-prefix`? The various debug refs mention "rulesPrefix" and "reverseRulesPrefix".

Reply:
I changed it to "rule-process" because I thought that better fit the added debug statements, that aren't really about the prefix.